### ISSUE TITLE
Moving a marker instead of rerendering when updating marker position

### ISSUE THREAD

**Issue by roshanmaind** [fullstackreact/google-maps-react#451]
I have a  map with certain markers on it. I have given the marker's position from a state variable and I am updating the state variable every 500 ms by a very small distance. But every time the position of the marker is updated the marker re-renders. Meaning that the original marker component is removed and a new marker is rendered on the new location. This causes a "blinking"/"flickering" effect on the marker when the distance between the current and new positions are too small.

Here's a [demo](https://codesandbox.io/s/hardcore-cache-hwbje?file=/src/MapComponent.js). Note how the marker appears to be blinking. 

Is there any way of making the position update of the marker smooth so that instead of re-rendering the marker to a new position causing that blinking it instead _moves_ to the new location?

**Comment by Graig123git** [comment]
@roshanmaind did you figure this out ?

**Comment by roshanmaind** [comment]
@Graig123git  Nope. I ended up using [a different npm module for google maps](https://www.npmjs.com/package/react-google-maps). That one does not have this issue.
